src/Components/NewTodo.js

```diff
@@ -123,39 +133,33 @@ render() {
         </select>
         <br />
         {userError && <p className="form__error">Choose a user</p>}
-
       </label>
-
-      <select
-        className="form__status"
-        onChange={this.handleStatus}
-      >
-        <option value="" hidden>Choose a status</option>
-        <option value="done">done</option>
-        <option value="in process">in process</option>
-      </select>
-      <br />
-
+      <label>
+        <select
+          value={completed}
+          className="form__status"
+          onChange={this.handleChangeStatus}
+        >
+          <option value="false">Choose a status</option>
+          <option value="done">done</option>
+          <option value="in process">in process</option>
```

Comment:
Попробуй поменять `done` и `in process` на 0 и 1
```suggestion
          <option value=1>done</option>
          <option value=0>in process</option>
```

Reply:
ПРобував і так, але не вийшло,вже пів дня не можу зробити цю штуку.Інше пофіксив